tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,170 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearDiffTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'python3'))
+        self.section.append(Setting('keywords', 'TODO'))
+        self.uut = KeywordBear(self.section, Queue())
+
+        self.annotation_bear_result_type = namedtuple('result', ['contents'])
+        self.dep_results = {'AnnotationBear': HiddenResult(
+            'AnnotationBear', {'comments': ()})}
+
+    def test_keyword(self):
```

Comment:
every test has a purpose, right? Ideally I'd rather split them up, just making everyone of them an own function